spec/octokit/client/repositories_spec.rb

```diff
@@ -200,6 +200,26 @@
         end
       end
     end # .branch_protection
+
+    describe ".topics", :vcr do
+      it "returns repository topics" do
+        topics = Octokit.topics(@repo.full_name, :accept => Octokit::Preview::PREVIEW_TYPES.fetch(:topics))
+        expect(topics.names).to include("octokit")
+        assert_requested :get, github_url("/repos/#{@repo.full_name}/topics")
+      end
+    end # .topics    
+
+    describe ".replace_all_topics", :vcr do
+      it "replaces all topics for a repository" do
+        new_topics = ["octocat", "github", "github-api"]
+        options = {
+          :accept => "application/vnd.github.mercy-preview+json"
```

Comment:
I missed this the first time around. Would you please access this through the `PREVIEW_TYPES` constant?

Reply:
Good catch!